package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "protractor-workshop-2020",
+  "version": "1.0.0",
+  "description": "This is a Workshop about Protractor",
+  "main": "index.js",
+  "scripts": {
+    "clean": "rm -rf dist",
+    "build": "npm run clean && tsc",
+    "test": "npm run build && protractor dist/protractor/local.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/SebastianUrbano/protractor-workshop-2020.git"
+  },
+  "keywords": [
+    "ui-testing",
+    "dojo",
+    "practice",
+    "protractor"
+  ],
+  "author": "[John Sebastian Urbano Lenis][john.urbano@correo.icesi.edu.co][https://github.com/SebastianUrbano]",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/SebastianUrbano/protractor-workshop-2020/issues"
+  },
+  "homepage": "https://github.com/SebastianUrbano/protractor-workshop-2020#readme",
+  "dependencies": {
+    "protractor": "^7.0.0"
```

Comment:
move as a dev dependency:  `npm install -D protractor`

Reply:
Ok, it's done thanks.